### PR TITLE
81 categories

### PR DESCRIPTION
**Pull request by luchogabutti**
Linear card;

https://linear.app/cuadernorojo/issue/CR-81/listado-de-categorias

### CODE REVIEW

File: src/screens/home/components/subreddits/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+              if (data != null)
+                setSubreddit(data.data.children);
+            });
+          }
+        )
+      }, []);
+
+    return (
+        <div className="subreddits-main">
+            <h2>Subreddits</h2  >
```
Comment (noelsp) [inline src/screens/home/components/subreddits/index.jsx]: fix `</h2> `

File: src/screens/home/components/subreddits/index.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            {subreddits && subreddits.map(subreddit => <Subreddit key={subreddit.id} subreddit={subreddit.data}/>)}
+        </div>
+    )
+}
+
+export default Subreddits
```
Comment (noelsp) [inline src/screens/home/components/subreddits/index.jsx]: Add end of file here

File: src/screens/home/components/subreddits/style.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+.subreddits-main {
+    background-color: #fff;
+    border-radius: 10px;
+    margin: 20px;
+    height: 2050px;
+}
+
+.subreddits-main h2 {
+    margin-left: 25px;
+}
```
Comment (noelsp) [inline src/screens/home/components/subreddits/style.css]: Add end of file

File: src/screens/home/index.jsx [hunk 4]
```diff
@@ -1,13 +1,19 @@
...
 import Articles from "./components/articles";
+import './style.css'
+import Subreddits from "./components/subreddits";
+
 
 const Home = () => {
 
     return  (
         <>
             <Header/>
```
Comment (noelsp) [inline src/screens/home/index.jsx]: `<Header />`

File: src/screens/home/index.jsx [hunk 5]
```diff
@@ -1,13 +1,19 @@
...
+            <div className="main">
             <Articles/>
```
Comment (noelsp) [inline src/screens/home/index.jsx]: Indentá y agregá `<Articles />`

File: src/screens/home/style.css [hunk 6]
```diff
@@ -0,0 +1,4 @@
+.main {
+    display: flex;
+    justify-content: center;
+    }
```
Comment (noelsp) [inline src/screens/home/style.css]: End of file: 

Adjust "}"
```
.main {
    display: flex;
    justify-content: center;
 }
```

File: src/screens/home/components/subreddits/index.jsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+                setSubreddit(data.data.children);
+            });
+          }
+        )
+      }, []);
+
+    return (
+        <div className="subreddits-main">
+            <h2>Subreddits</h2  >
+            {subreddits && subreddits.map(subreddit => <Subreddit key={subreddit.id} subreddit={subreddit.data}/>)}
```
Comment (noelsp) [inline src/screens/home/components/subreddits/index.jsx]: Deconstruct the map param: 

`{subreddits && subreddits.map(({id, data}) => <Subreddit key={id} subreddit={data}/>)}`

**Review by noelsp — CHANGES_REQUESTED** [review]

File: src/screens/home/components/subreddits/index.jsx (hunk 1 above)
Comment (luchogabutti) [reply src/screens/home/components/subreddits/index.jsx]: done

**Review by noelsp — APPROVED** [review]

**Pull request merged**